ghost/core/core/server/api/endpoints/tiers.js

```diff
@@ -100,23 +63,11 @@ module.exports = {
         headers: {
             cacheInvalidate: true
         },
-        validation: {
```

Comment:
Should we be keeping `id` as an `option` here? Or is it actually optional now?

Reply:
No that's my B - will fix